src/interactive-window/editor-integration/codewatcher.ts

```diff
@@ -483,7 +481,7 @@ export class CodeWatcher implements ICodeWatcher {
         }
     }
 
-    @captureTelemetry(Telemetry.SelectCellContents)
+    @capturePerfTelemetry(Telemetry.SelectCellContents)
```

Comment:
I think these were all meant to be just capturing the usage, however we've been sending the total durtation it took to execute these methods as well
Now that we have individual owners, we can have each owner review and change these accordingly